### DISCUSSION TITLE
queue.playing broken?

### DISCUSSION THREAD

**Discussion by AlohaSteam** [Androz2091/discord-player#1159] [Q&A] [answered]
No idea if this is me or what but the queue.playing function is not working. It just seems as it completely disregards it, now I'm learning js so idk if its just me doing something wrong or if its the actual package itself.

`
        const queue = player.createQueue(message.guild, {
            metadata: VoiceChannel
        })

        if(!queue.playing) {
            queue.play(song);
            message.channel.send(`${song.title} is now playing.`)
        } else if(queue.playing) {
            queue.addTrack(song);
            message.channel.send(`${song.title} is queud up to play.`)
        }
`

from the documentation and the source code from the bots that use discord-player this should work right?

**Comment by jarrodback** [comment]
Yeh it seems broken for me too. In discord-playert/dist/Structures/Queue.js line 289 function setPaused
add:
`this.playing = !paused`
after both the if statements. Should temp fix it for now

**Comment by febkosq8** [comment, marked as answer]
For those wondering, check `queue.connection.paused` to see if the player is paused.
`queue.playing` always stays true.
